### DISCUSSION TITLE
LC-Relay-ESP01-2R-5V goes on & off in less that 1 seconde

### DISCUSSION THREAD

**Discussion by gruikster** [arendst/Tasmota#15563] [Q&A] [answered]
Hi all,
I'm newbie on Tasmota world.
I can remote control a LC-Relay-ESP01-2R-5V with Tasmota via my WiFi but when I press ON button (to action the relay),
it goes ON and immediately OFF.
I tried many different template and reconfigured many time the ESP01.
But still the same.

- See console display when I click ON:
12:35:23.963 RSL: RESULT = {"POWER1":"ON"}
12:35:23.965 RSL: POWER1 = ON
12:35:24.039 RUL: POWER1#STATE=1 performs "SerialSend5 A00101A2"
12:35:24.043 RSL: RESULT = {"SerialSend":"Done"}
12:35:24.389 RSL: RESULT = {"SerialReceived":"xxxxxxxxx"}
12:35:25.278 RSL: RESULT = {"SerialReceived":"xxxxxxxx"}
12:35:25.695 RSL: RESULT = {"SerialReceived":"xxxxxxxxx"}
12:35:27.009 RSL: RESULT = {"SerialReceived":"xxxxxxxxx"}
12:35:27.730 RSL: RESULT = {"SerialReceived":"xxxxxxxx"}
12:35:28.328 RSL: RESULT = {"SerialReceived":"xxxxxxxx"}+

- Even when I type directly on the console the command :  "SerialSend5 A00101A2 endon"
it's the same.

Anyone encountered this issue ?

Thx in advance for your help.

Rgds

**Comment by barbudor** [comment, marked as answer]
May be not enough power ?
As soon as the relay is driven ON, the power fails ?

**Reply by gruikster** [reply]
OMG, you're right !
I tried without being on the USB of the PC but with an external power supply and it's ok :)
Thx for your input, 
I didn't realized that this could eat so much power.

best regards
